templates/ts-apollo-postgres-backend/src/index.ts

```diff
@@ -4,19 +4,50 @@ import { ApolloServer } from "apollo-server-express"
 import cors from "cors"
 import express from "express"
 import http from "http"
-import { printSchema } from 'graphql'
-import { createRuntime } from './runtime'
+import { buildGraphbackAPI } from 'graphback'
+import { loadSchemaSync } from '@graphql-tools/load'
+import { GraphQLFileLoader } from '@graphql-tools/graphql-file-loader'
+import path from 'path'
+import { loadDBConfig } from './db'
+import { migrateDB, removeNonSafeOperationsFilter } from 'graphql-migrations'
+import { createKnexDbProvider } from '@graphback/runtime-knex'
+import { ClientCRUDPlugin } from '@graphback/codegen-client'
 
 const app = express()
 
 app.use(cors())
 
-// connect to db
-const { schema, resolvers } = createRuntime();
+const modelDefs = loadSchemaSync(path.resolve('./model/*.graphql'), {
+  loaders: [
+    new GraphQLFileLoader()
+  ]
+})
+
+const dbConfig = loadDBConfig()
+
+const { typeDefs, resolvers, services } = buildGraphbackAPI(modelDefs, {
+  dataProviderCreator: createKnexDbProvider(dbConfig),
+  plugins: [
+    new ClientCRUDPlugin({
```

Comment:
This makes no sense in this context. we not going to generate client queries at runtime.

Config need to stay to use cli to generate client side. we will think how to transform that outside this PR

Reply:
Cool